tests/test_il_pollution_control.py

```diff
@@ -0,0 +1,95 @@
+from datetime import datetime
+from os.path import dirname, join
+
+import pytest
+from city_scrapers_core.constants import BOARD, CANCELLED, PASSED, TENTATIVE
+from city_scrapers_core.utils import file_response
+from freezegun import freeze_time
+
+from city_scrapers.spiders.il_pollution_control import IlPollutionControlSpider
+
+test_minutes_response = file_response(
```

Comment:
Agreed that this can be removed and the `il_pollution_control.html` file can be deleted

Reply:
Done!